esp-hal-procmacros/Cargo.toml

```diff
@@ -41,3 +41,9 @@ is-lp-core = []
 ## Provide an `#[entry]` macro for running applications on the ESP32-S2/S3's
 ## ULP core.
 is-ulp-core = []
+
+## Suport RISC-V architecture
+riscv = []
```

Comment:
Are these used/required?

Reply:
oops - I thought I removed them before the commit 🤦‍♂️